### PR TITLE
fix(samples): lro added out-of-order step.

### PR DESCRIPTION
**Pull request by dfirova**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java [hunk 1]
```diff
@@ -56,6 +59,13 @@ public static void addFulfillmentPlaces(String productName, String placeId)
             .setAllowMissing(true)
             .build();
 
+    // To send an out-of-order request assign the invalid AddTime here:
+    // Timestamp previousDay = Timestamp.newBuilder()
+    //        .setSeconds(Instant.now().minusSeconds(86400).getEpochSecond())
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java:64]: Can we do minusDays?

File: samples/interactive-tutorials/src/main/java/product/RemoveFulfillmentPlaces.java [hunk 2]
```diff
@@ -57,6 +57,13 @@ public static void removeFulfillmentPlaces(String productName, String storeId)
             .setAllowMissing(true)
             .build();
 
+    // To send an out-of-order request assign the invalid AddTime here:
+    // Timestamp previousDay = Timestamp.newBuilder()
+    //        .setSeconds(Instant.now().minusSeconds(86400).getEpochSecond())
+    //        .setNanos(Instant.now().minusSeconds(86400).getNano())
+    //        .build();
+    // removeFulfillmentRequest = removeFulfillmentRequest.toBuilder().setAddTime(previousDay).build();
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/product/RemoveFulfillmentPlaces.java:65]: setRemoveTime

File: samples/interactive-tutorials/src/main/java/product/SetInventory.java [hunk 3]
```diff
@@ -93,6 +93,13 @@ public static void setInventory(String productName) throws IOException, Interrup
             .build();
     System.out.printf("Set inventory request: %s%n", setInventoryRequest);
 
+    // To send an out-of-order request assign the invalid AddTime here:
+    // Timestamp previousDay = Timestamp.newBuilder()
+    //        .setSeconds(Instant.now().minusSeconds(86400).getEpochSecond())
+    //        .setNanos(Instant.now().minusSeconds(86400).getNano())
+    //        .build();
+    // setInventoryRequest = setInventoryRequest.toBuilder().setAddTime(previousDay).build();
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/product/SetInventory.java:101]: setSetTime

**Review by chenlei1216 — APPROVED** [review]

File: samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java (hunk 1 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java]: With Instant and protobuf timestamp only minusSeconds, so I added an instant variable, created via LocalDateTime, which look better then having minus seconds.

File: samples/interactive-tutorials/src/main/java/product/RemoveFulfillmentPlaces.java (hunk 2 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/product/RemoveFulfillmentPlaces.java]: Done

File: samples/interactive-tutorials/src/main/java/product/SetInventory.java (hunk 3 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/product/SetInventory.java]: Done

**Review by Sita04 — APPROVED** [review]
LGTM. Thanks for the changes.

I am a shadow reviewer. Tagging @kurtisvg to take a look at the PR.

**Comment by dfirova** [comment]
@Sita04 merged changes from main, please rerun the tests

File: samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java [hunk 4]
```diff
@@ -56,6 +56,15 @@ public static void addFulfillmentPlaces(String productName, String placeId)
             .setAllowMissing(true)
             .build();
 
+    // To send an out-of-order request assign the invalid AddTime here:
+    // Instant instant = LocalDateTime.now().minusDays(1).toInstant(ZoneOffset.UTC);
+    // Timestamp previousDay = Timestamp.newBuilder()
```
Comment (kurtisvg) [inline samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java:61]: We strongly prefer not having commented out code, as it's not tested or compiled.

**Review by kurtisvg — APPROVED** [review]

**Pull request merged**